src/checkout/checkout-store-selector.ts

```diff
@@ -147,6 +147,17 @@ export default interface CheckoutStoreSelector {
      */
     getSelectedPaymentMethod(): PaymentMethod | undefined;
 
+    /**
+     * Gets the available flash messages.
+     *
+     * Flash messages contain messages set by the server,
+     * e.g: when trying to sign in using an invalid email link.
+     *
+     * @param type - The type of flash messages to be returned. Optional
+     * @returns The flash messages if available, otherwise undefined.
+     */
+    getFlashMessages(type?: FlashMessageType): FlashMessage[] | undefined;
```

Comment:
I think if you want to import `FlashMessageType` and use it as a value (not a `type` or interface). You'll need to explicitly export it in `bundles/checkout-sdk.ts`. Otherwise, you can't use it in the client app and you'll need to put in `0`, `1` etc... into this method.

Reply:
I have created a build and this seems to work fine. Managed to use in UCO without issues:

<img width="675" alt="Screen Shot 2020-05-11 at 2 21 05 pm" src="https://user-images.githubusercontent.com/1621894/81523919-ba665b80-9392-11ea-9be3-80e3d14b7625.png">

Reply:
Hrm, interesting. I might be wrong but I think if you actually try to run the code in the browser, you'll get an error. This is because the enum is actually not exported to the `js` file (only in the `d.ts` file).